gammapy/irf/psf_table.py

```diff
@@ -2,8 +2,9 @@
 import logging
 import numpy as np
 from scipy.interpolate import UnivariateSpline
+from scipy.integrate import cumtrapz
 from astropy.io import fits
-from astropy.units import Quantity
+from astropy import units as u
```

Comment:
Is using `u.Quantity` instead of importing `Quantity` preferable for some reason?

Reply:
`from astropy import units as u` is the recommended import pattern for `astropy.units`, which we already use in most parts of Gammapy. To avoid the second import I typically use `u.Quantity()`. It doesn't really matter, but slightly prefer, when it's consistent.